ps3mf/save.go

```diff
@@ -16,6 +17,10 @@ const (
 	fdmSupportsPaintingVersion = "1"
 )
 
+var CurrentDate = func() string {
```

Comment:
Function should be named with a verb to indicate it's a function, e.g. `GetCurrentDate`

Reply:
Also, no need to use a `var` for this:

```suggestion
func GetCurrentDate() string {
```